telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -1572,14 +1585,16 @@ class TelnyxClient(
             // Set global callID
             callId = offerCallId
 
-
             peerConnection = Peer(
                 context, client, providedTurn, providedStun, offerCallId.toString(),
                 object : PeerConnectionObserver() {
                     override fun onIceCandidate(p0: IceCandidate?) {
                         super.onIceCandidate(p0)
-                        peerConnection?.addIceCandidate(p0)
-                        Timber.d("On IceCandidate Added")
+                        Timber.d("Event-IceCandidate Generated")
+                        if (calls[callId]?.getCallState()?.value != CallState.ACTIVE){
```

Comment:
Please add a space between parentheses and braces` ){`.

And check if peerConnection Exist before printing the log